packages/vitest-pool-workers/src/pool/index.ts

```diff
@@ -20,6 +20,7 @@ import {
 	testRegExps,
 	WebSocket,
 } from "miniflare";
+import semverSatisfies from "semver/functions/satisfies.js";
```

Comment:
Why is there a `.js` on the end of this module specifier?

Reply:
I'm not entirely sure, but I assume it's just due to how `semver` is built and published in a very CJS fashion?

Without it, I got an error like this:
```
Error: Cannot find module 'E:\xxx\node_modules\semver\functions\satisfies' imported from E:\xxx\node_modules\@cloudflare\vitest-pool-workers\dist\pool\index.mjs
Did you mean to import semver/functions/satisfies.js?
 ❯ finalizeResolution node:internal/modules/esm/resolve:264:11
 ❯ moduleResolve node:internal/modules/esm/resolve:917:10
 ❯ defaultResolve node:internal/modules/esm/resolve:1130:11
 ❯ ModuleLoader.defaultResolve node:internal/modules/esm/loader:396:12
 ❯ ModuleLoader.resolve node:internal/modules/esm/loader:365:25
 ❯ ModuleLoader.getModuleJob node:internal/modules/esm/loader:240:38
 ❯ ModuleWrap.<anonymous> node:internal/modules/esm/module_job:85:39
 ❯ link node:internal/modules/esm/module_job:84:36
```

And when swapping over to `semver/functions/satisfies.js` everything was happy, so I didn't dig any further.

Reply:
Another option would be to just `import semver from 'semver'` and then using `semver.satisfies`, if that's preferred. Let me know.